influxdb_client/client/_base.py

```diff
@@ -60,6 +60,10 @@ def __init__(self, url, token, debug=None, timeout=10_000, enable_gzip=False, or
         self.conf.enable_gzip = enable_gzip
         self.conf.verify_ssl = kwargs.get('verify_ssl', True)
         self.conf.ssl_ca_cert = kwargs.get('ssl_ca_cert', None)
+        self.conf.cert_file = kwargs.get('cert_file', None)
+        self.conf.key_file = kwargs.get('key_file', None)
+        self.conf.key_password = kwargs.get('key_password', None)
+        self.conf.ssl_context = kwargs.get('ssl_context', None)
```

Comment:
Document this options in: `InfluxDBClient.__init__` and `InfluxDBClientAsync.__init__`.

- https://github.com/influxdata/influxdb-client-python/blob/5ba25c5529109d6afdc4eeadc456a4444ccd7dfc/influxdb_client/client/influxdb_client.py#L27
- https://github.com/influxdata/influxdb-client-python/blob/5ba25c5529109d6afdc4eeadc456a4444ccd7dfc/influxdb_client/client/influxdb_client_async.py#L19

Reply:
I've documented the corresponding options and renamed all mTLS-related variables transparently and centrally.